custom_components/tuya_local/devices/Fresco_Hydrate_Ultra_Pet_Fountain.yaml

```diff
@@ -0,0 +1,111 @@
+name: Hydrate Ultra
+products:
+  - id: cuw2k3ho3e25fade
+    name: Hydrate Ultra
+
+primary_entity:
+  entity: light
+  icon: "mdi:water-pump"
+  dps:
+    - id: 101
+      type: boolean
+      name: switch
+    - id: 102
+      name: brightness
+      type: integer
+      range:
+        min: 1
+        max: 10
+
+secondary_entities:
+  - entity: button
+    name: Reset Filter Alarm
+    icon: "mdi:air-filter"
+    category: config
+    dps:
+      - id: 104
+        type: integer
+        name: button
+  - entity: sensor
+    name: Filter life
+    icon: "mdi:air-filter"
+    category: diagnostic
+    dps:
+      - id: 104
+        type: integer
+        name: sensor
+        precision: 0
+        range:
+          min: 0
+          max: 43200
+        mapping:
+          - scale: 1440
+            invert: true
+        unit: d
+  - entity: button
+    name: Reset Pump Alarm
+    icon: "mdi:pump"
+    category: config
+    dps:
+      - id: 105
+        type: integer
+        name: button
+  - entity: sensor
+    name: Pump maintenance
+    icon: "mdi:pump"
+    category: diagnostic
+    dps:
+      - id: 105
+        type: integer
+        name: sensor
+        precision: 0
+        range:
+          min: 0
+          max: 86400
+        mapping:
+          - scale: 1440
+            invert: true
+        unit: d
+  - entity: sensor
+    name: Water quality
+    icon: "mdi:water"
+    category: diagnostic
+    dps:
+      - id: 106
+        type: integer
+        name: sensor
+        unit: TDS
+  - entity: switch
+    name: UV sterilization
+    icon: "mdi:flash"
+    category: config
+    dps:
+      - id: 107
+        type: boolean
+        name: switch
+  - entity: sensor
+    name: Water level
+    icon: "mdi:water"
+    category: diagnostic
+    dps:
+      - id: 108
+        type: integer
+        name: sensor
+        mapping:
+          - dps_val: "1"
+            value: 1
+          - dps_val: "2"
+            value: 2
+          - dps_val: "3"
+            value: 3
+          - dps_val: "4"
+            value: 4
+  - entity: sensor
+    name: UV runtime
+    icon: "mdi:timer-outline"
+    category: diagnostic
+    dps:
+      - id: 109
+        type: integer
+        name: sensor
+        unit: s
```

Comment:
Add `class: duration` to the entity config to have HA automatically display the appropriate units for the scale of the value.

Reply:
I can do that, but than you will need to update most of the last added fountains or feeders.
As that part is a copy of your own template that i used.

Its a bit more work if there is no up to date info.

Reply:
Updating old configs that may have done things imperfectly in the past is a separate issue from accepting new changes into the repository. Please don't insist that the other 600 configs, some dating back 5 years before things like the duration class and translation_key were available in HA, be updated before you will follow my advice for a better quality contribution, as that will only result in the PR being put on hold.

Reply:
An example is this Sobralik Pet Water Fountain (still open), i used that template.. 


I maybe can fix https://github.com/make-all/tuya-local/pull/1846  that one too.